Task2/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam.esm'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+ext {
+    junitVersion = '5.7.1'
+    springVersion = '5.3.5'
+}
+
+subprojects { project ->
+    project.apply plugin: 'java'
+
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation group: 'org.springframework', name: 'spring-webmvc', version: '5.3.5'
```

Comment:
@Pokemon3108 My suggestion to use such view `implementation "org.apache.commons:commons-lang3:${rev_apache_commons}"`